grammars/asciidoc.cson

```diff
@@ -84,6 +206,32 @@
     ]
   }
 
+  # Matches explicit paragraph styles
+  #
+  # Examples
+  #
+  #   [normal]
+  #   This is a normal paragraph.
+  {
+    'match': '^\\[((normal|literal|listing|TIP|NOTE|IMPORTANT|WARNING|CAUTION|partintro|comment|example|sidebar|source|music|latex|graphviz))\\]$'
```

Comment:
I thought about that, too. I chose to match just these items because they are the only ones in the specification. By identifying these specifically, the highlighting could help authors detect typos.